Studio/src/Interface/ShapeWorksStudioApp.cpp

```diff
@@ -1754,6 +1771,18 @@ void ShapeWorksStudioApp::compute_mode_shape() {
   visualizer_->display_shape(analysis_tool_->get_mode_shape(pca_mode, pca_value));
 }
 
+//---------------------------------------------------------------------------
+void ShapeWorksStudioApp::compute_mca_mode_shape() {
```

Comment:
It seems redundant to have `compute_mca_mode_shape` and `compute_mode_shape` as the former will do what the latter does when the mode is `vanilla`.